synapse/config/registration.py

```diff
@@ -330,6 +334,22 @@ def generate_config_section(self, generate_secrets=False, **kwargs):
             #email: https://example.com     # Delegate email sending to example.com
             #msisdn: http://localhost:8090  # Delegate SMS sending to this local process
 
+        # If false, don't let users set their own display names/avatars
+        # (unless they are a server admin) other than for the very first time.
+        # Useful when provisioning users based on the contents of a 3rd party
+        # directory and to avoid ambiguities.
+        # Default is true and users can edit their own display names/avatars.
+        #
+        #enable_set_displayname: false
+        #
+        #enable_set_avatar_url: false
```

Comment:
Mm, could we reorganise this so that there's one bit of explanation per option like every other option?

Something like:

```
        # Whether users are allowed to change their displayname after it has
        # been initially set. Useful when provisioning users based on the
        # contents of a third-party directory.
        #
        # Does not apply to server administrators. Defaults to 'true'
        #
        #enable_set_displayname: false

        # Whether users are allowed to change their avatar after it has been
        # initially set. Useful when provisioning users based on the contents
        # of a third-party directory.
        #
        # Does not apply to server administrators. Defaults to 'true'
        #
        #enable_set_avatar_url: false
```